x-pack/plugins/ml/server/routes/results_service.ts

```diff
@@ -327,7 +329,7 @@ export function resultsServiceRoutes({ router, routeGuard }: RouteInitialization
   /**
    * @apiGroup ResultsService
    *
-   * @api {get} /api/ml/results/category_stopped_partitions
+   * @api {get} /api/ml/results/category_stopped_partitions Get partitions that have stopped being categorized
```

Comment:
this one is actually `POST`
```suggestion
   * @api {post} /api/ml/results/category_stopped_partitions Get partitions that have stopped being categorized
```

Reply:
Done in https://github.com/elastic/kibana/pull/111298/commits/2148278afae5c49dbed7850e9de6678792cb0434